cloud/openstack/os_floating_ip.py

```diff
@@ -76,6 +76,12 @@
      choices: [present, absent]
      required: false
      default: present
+   purge:
+     description:
+        - When I(state) is absent, indicates whether or not to delete the floating
+          IP completely, or only detach it from the server. Default is to detach only.
+     required: false
+     default: false
```

Comment:
Completely disagree here. We cannot change the default behaviour as others may already be depending on it.